vars/commonPipelineEnvironment.groovy

```diff
@@ -25,11 +41,35 @@ class commonPipelineEnvironment implements Serializable {
             return configProperties[property]
     }
 
+    def setInfluxCustomData(Map myData = [:]) {
```

Comment:
1.) It is hard to understand what is the difference between influxCustomData and influxCustomDataMap, especially since both methods deal with a map. Maybe we should clarify the difference with a better naming.
2.) Is there a real use case for replacing the existing map, and dropping any already collected values?

Reply:
I removed the setters for `influxCustomData` and `influxCustomDataMap`.